spec.html

```diff
@@ -167,15 +167,16 @@ <h1>The ElementDescriptor Specification Type</h1>
       <emu-table id="table-element-descriptor-fields" caption="ElementDescriptor fields">
         <table>
           <thead>
-            <tr> <th>Field Name</th>      <th>Value</th>                                                          </tr>
+           <tr> <th>Field Name</th>                        <th>Value</th>                                                          </tr>
```

Comment:
not to be too picky, but these should be indented by two spaces, not one :-)

Reply:
Oh, i see. I will fix it after school :)